documentation/book/proc-deploying-the-topic-operator-standalone.adoc

```diff
@@ -0,0 +1,63 @@
+
+[id='deploying-the-topic-operator-standalone']
+= Deploying the Topic Operator standalone
+
+Deploying the Topic Operator as a standalone component is more complicated than installing it using the Cluster Operator, but is more flexible.
+For instance is can operate _with_ any Kafka cluster, not necessarily one deployed by the Cluster Operator.
+
+.Prerequisites
+
+* An existing Kafka cluster for the Topic Operator to connect to.
+
+.Procedure
+
+. Edit the `examples/install/cluster-operator/05-Deployment-strimzi-topic-operator.yaml` resource. You will need to change the following
++
+* The `STRIMZI_KAFKA_BOOTSTRAP_SERVERS` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of bootstrap brokers in your Kafka cluster, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs.
+* The `STRIMZI_ZOOKEEPER_CONNECT` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to a list of the Zookeeper nodes, given as a comma-separated list of `_<hostname>_:‍_<port>_` pairs. This should be the same Zookeeper cluster that your Kafka cluster is using.
+* The `STRIMZI_NAMESPACE` environment variable in `Deployment.spec.template.spec.containers[0].env` should be set to the {ProductPlatformName} namespace in which you want the operator to watch for  `KafkaTopic` resources.
+
+. Deploy the Cluster Operator.
++
+ifdef::Kubernetes[]
+On {KubernetesName} this can be done using `kubectl apply`:
++
+[source,shell]
+kubectl apply -f examples/install/cluster-operator
++
+On {OpenShiftName} this can be done using `oc apply`:
```

Comment:
I think this is in the wrong place and says the same as line 31.

Reply:
When we're documenting both the sentence introducing each command example is "On XXX this can be done using YYY". When documenting just OS it would be weird to say "On OpenShift...", so I use a `ifndef` around line 31 so that sentence is just "This can be done using `oc xyz`".

Reply:
Do we have the ifs in the docu more complicated than in the code now? I get what it does now. But it will be hard to maintain and quite easy to get wrong with one of the next updates.

Reply:
Honestly, I think the easiest way to maintain this in the medium to long term is to generate the docs twice upstream, once for Kubernetes and once for OpenShift. I think that's the only way we will be able to generate quality documentation in both cases. Otherwise we're going to be stuck with these `ifdef`s all over the place. I think this is good enough in the immediate term.